test/release/examples/primers/locales.chpl

```diff
@@ -201,88 +201,38 @@ writeln();
 
 
 //
-// Next, let's create a simple distributed data structure using the
-// principles above.  In particular, we'll create a linked list that
-// spans the locales with a list node per locale.
-//
-
-//
-// First, we'll define a standard ``Node`` class for linked lists.  Note
-// that even though the ``next`` fields will span locales, they don't need
-// to be declared specially in any way, again thanks to the [P]GAS
-// nature of the language.
-//
-class Node {
-  var data: real;
-  var next: unmanaged Node;
-}
-
+// Class instances are considered to be wherever the fields are
+// stored, rather than where the variable referring to the instance
+// is stored.
+// This rule applies to non-nil variables of borrowed or unmanaged
+// class type.
 
-//
-// Next, we'll iterate through the locales, allocating a node on each.
-// It's worth reviewing that a class variable (like the ``next`` field
-// above) is just a reference in Chapel and the object it points to
-// is a separate thing allocated by a call to ``new``.  So in general,
-// the reference and object can live on different locales (as they
-// will in our example).
-//
-// So, in the following loop, ``current`` will always point at the last
-// node we created, while our on-clause will move the task ahead to
-// the next locale.  That way, our execution of ``new`` will create
-// the new object on that "next" locale.
-//
-var head    = new unmanaged Node(0);
-
-var current = head;
-
-for i in 1..numLocales-1 do
-  on Locales[i] {
-    current.next = new unmanaged Node(i);
-    current      = current.next;
-  }
+// Let's explore this idea by creating a class instance and
+// querying its locality.
 
-//
-// Let's loop over the linked list and output each node and the locale on
-// which it exists using ``.locale.id`` to determine the locale.  This
-// computation takes place entirely on locale 0 and accesses remote
-// memory as necessary.
-//
-current = head;
-
-while current {
-  writeln("node with data = ", current.data, " on locale ", current.locale.id);
-  current = current.next;
+class Data {
+  var x:int;
 }
 
-writeln();
+var myData:unmanaged Data; // myData is a pointer-to-nil stored on Locale 0
 
-//
-// Alternatively, we could use a data-driven execution to walk the
-// linked list.  Note that on-clauses, when applied to class
-// variables, resolve to the locale of the object, not the reference.
-//
-current = head;
+on Locales[1 % numLocales] {
+  writeln("at start of on 1, myData is on locale ", myData.locale.id);
```

Comment:
I would separate the English sections of this output from the Chapel syntax that is being referred to.  E.g. "at the start of 'on 1', myData is on locale ".  I would do something similar for the next couple of writelns